src/gluonnlp/embedding/evaluation.py

```diff
@@ -32,12 +32,10 @@
 
 class _WordEmbeddingEvaluationFunction(HybridBlock):  # pylint: disable=abstract-method
     """Base class for word embedding evaluation functions."""
-    pass
```

Comment:
Why should we remove the `pass` here?

Reply:
It's not required. Lint fails (pylint 2 introduced a new rule)